src/character.cpp

```diff
@@ -1375,9 +1375,20 @@ float Character::stability_roll() const
 
 bool Character::is_dead_state() const
 {
-    return get_part_hp_cur( body_part_head ) <= 0 ||
-           get_part_hp_cur( body_part_torso ) <= 0 ||
-           is_dead;
+    // we want to warn the player with a debug message if they are invincible. this should be unimportant once wounds exist and bleeding is how you die.
+    bool has_vitals = false;
+    for( const bodypart_id &part : get_all_body_parts( get_body_part_flags::only_main ) ) {
+        if( part->is_vital ) {
+            if( get_part_hp_cur( part ) <= 0 ) {
+                return true;
+            }
+            has_vitals = true;
+        }
+    }
+    if( !has_vitals ) {
+        debugmsg( _( "WARNING!  Player has no vital part and is invincible." ) );
```

Comment:
Shouldn't the player die instead? If a spell remove your Head and your Torso you should be dead instead of immortal I think. Or is that just a placeholder because by the time this is finished `is_vital` will go away anyway?

Reply:
i'm not really sure. really it could go either way but because it's a bug i'd rather it not kill your character